std_msgs/Header header

float32 wavelength
float32 measurement_distance

float32 transmittance
float32 variance
